### PR TITLE
Fix help pane scrollbar code background color

### PR DESCRIPTION
**Pull request by sharon-wang**
### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

Addresses the code background color bug in #11520.

#### Dark Theme Fix Preview
image

#### Light Theme Fix Preview
image

### Approach
Sets the `background-colour` of the `div.sourceCode` to the light or dark code background color, depending on the active theme. The `border-radius` is also set to the same value as `pre`, otherwise the code box loses the rounded corners.

**Note:** The code background grey color is defined in `HelpPaneEditorStyles.css` and there is one grey used for all dark themes and another grey used for all light themes. It's possible the grey could be incompatible with some of the dark or light themes from an accessibility standpoint. Should a code background color be defined for each theme?

### Automated Tests
N/A?

### QA Notes

- Please test with a light editor theme and a dark editor theme
- Please test with both `Modern` and `Sky` RStudio themes for light and dark editor themes
- Other code areas outside of the help panel should not be affected by this change

### Documentation
N/A

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [ ] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/workbench/views/help/HelpPaneEditorStyles.css [hunk 1]
```diff
@@ -183,5 +185,20 @@ p + div[style="text-align: center;"]
...
+/**
+ * This styling ensures the sourceCode area has a solid, continuous
+ * background color, even if the view is narrowed and a scrollbar
+ * appears to allow for horizontal scrolling.
+ */
+ .ace_editor_theme div.sourceCode {
+   border-radius: 3px;
+}
+
+.rstudio-themes-default.ace_editor_theme div.sourceCode {
```
Comment (timtmok) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/help/HelpPaneEditorStyles.css:197]: This has a slightly different look in light mode for some editor themes. When switching to `Chrome` or `Eclipse` for example, it doesn't cover the entire background with the colour. It seems that there's no `rstudio-themes-default` for these themes but there is a `rstudio-themes-light` class on it.

image

Comment (sharon-wang) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/help/HelpPaneEditorStyles.css]: For some reason `rstudio-themes-default` shows up for the themes `Chrome` and `Eclipse` on my end (and the background colour is covered), but I can use `rstudio-themes-light-menus` instead which also works for light themes on my end.

I've also changed the dark theme rule to use `rstudio-themes-dark-menus` instead, so it's similar to the light theme rule. I tested with various light and dark themes and seems to be working for me. Let me know if you see different behaviour!

Comment (timtmok) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/help/HelpPaneEditorStyles.css]: I tried out some different settings and `rstudio-themes-default` is tied to the Appearance -> RStudio Theme setting with the `Modern` value. Changing it to `Sky` will use `rstudio-themes-alternate`.

It feels kind of weird to use `rstudio-themes-[light|dark]-menus` for the selector. I think a CSS OR selector would work better here to handle the different classes used to represent `Modern` and `Sky`.

**Review by timtmok — APPROVED** [review]
Good work!

**Pull request merged**